<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>generate_qr_panel</class>
 <widget class="QWidget" name="generate_qr_panel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GENERATE QR</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(213, 145, 110);
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Reem Kufi&quot;;</string>
   </property>
   <property name="text">
    <string>GENERATE QR CODES</string>
   </property>
  </widget>
  <widget class="QPushButton" name="HUCK_BTN">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-radius: 50px;
}

QPushButton:enabled {
   background-color: #d5916e;
}

QPushButton:pressed {
   background-color: #CE8865;

}

QPushButton:hover:!pressed {
   background-color: #eac4a3;
	border-radius: 50px;
}

QPushButton:disabled {
   background-color: #d5916e;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../Downloads/envelope.png</normaloff>../../../Downloads/envelope.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>130</width>
     <height>130</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Reem Kufi&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Send QR to</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Reem Kufi&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>all Home Owners</string>
   </property>
  </widget>
  <widget class="QPushButton" name="HUCK_BTN_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>180</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-radius: 50px;
}

QPushButton:enabled {
   background-color: #d5916e;
}

QPushButton:pressed {
   background-color: #CE8865;

}

QPushButton:hover:!pressed {
   background-color: #eac4a3;
	border-radius: 50px;
}

QPushButton:disabled {
   background-color: #d5916e;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../Downloads/gift-card.png</normaloff>../../../Downloads/gift-card.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>130</width>
     <height>130</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="HUCK_BTN_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-radius: 50px;
}

QPushButton:enabled {
   background-color: #d5916e;
}

QPushButton:pressed {
   background-color: #CE8865;

}

QPushButton:hover:!pressed {
   background-color: #eac4a3;
	border-radius: 50px;
}

QPushButton:disabled {
   background-color: #d5916e;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../Downloads/paper-plane.png</normaloff>../../../Downloads/paper-plane.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>130</width>
     <height>130</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>290</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Reem Kufi&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Send QR to</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>320</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Reem Kufi&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>a Visitor</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>290</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Reem Kufi&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Send QR to</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>320</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Reem Kufi&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>a Selected User</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>400</y>
     <width>411</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QProgressBar {
        border: 2px solid rgba(33, 37, 43, 180);
        border-radius: 5px;
        text-align: center;
        background-color: rgba(33, 37, 43, 180);
        color: black;
    }
    QProgressBar::chunk {
       
	background-color: rgb(234, 196, 163);
    }</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
